src/lib/seam/components/AccessCodeTable/AccessCodeTable.tsx

```diff
@@ -229,6 +226,19 @@ export function AccessCodeTable({
           disableDeleteAccessCode={disableDeleteAccessCode}
         />
       </TableBody>
+
+      {isError && (
+        <Snackbar
+          message={
+            error?.message ??
+            'An unknown error occurred fetching access codes. Please try again.'
```

Comment:
Alternatives:

- `Unknown error fetching X. Please try again.`
- `Unexpected error fetching X. Please try again.`
- `An unknown error occurred fetching X. Please reload.`
- `Something went wrong. Please try again`.

Reply:
Was there a message suggested on the design? What about `loading` instead of `fetching`?

Reply:
Updated with design recommendation